app/src/main/java/de/danoeh/antennapod/dialog/EpisodesApplyActionFragment.java

```diff
@@ -451,10 +532,18 @@ private void deleteChecked() {
                 DBWriter.deleteFeedMediaOfItem(getActivity(), episode.getMedia().getId());
             }
         }
-        close();
+        close(R.plurals.deleted_episode_batch_label, checkedIds.size());
     }
 
-    private void close() {
+    private void close(@PluralsRes int msgId, int numItems) {
+        if (numItems > 0) {
+            Snackbar.make(getActivity().findViewById(R.id.content),
+                    getResources().getQuantityString(msgId, numItems, numItems),
+                    Snackbar.LENGTH_LONG
+                    )
+                    .setAction(android.R.string.ok, v -> {})
+                    .show();
```

Comment:
I suggest to remove the action button and use `LENGTH_SHORT` instead

Reply:
I did start with `LENGTH_SHORT`. When I tested it, I found that often the message was gone, by the time I got around to try to read it.